samples/snippets/src/main/java/com/example/spanner/HubbleTransactionsCodeLab.java

```diff
@@ -203,47 +242,34 @@ public void writeMessagesUUID(
                 .set("msg_id")
                 .to(UUID.randomUUID().toString())
                 .set("body")
```

Comment:
Do we need to update the message subject?

Reply:
Will do

Reply:
Is this done or still pending?

Reply:
Based on my understanding of this comment I have done it